### PR TITLE
Terraform configs for test site, simple Jenkins pipeline for deployments

### PR DESCRIPTION
**Pull request by rnagle**
Adds some terraform configs to manage an S3 bucket for hosting a test version of the site. Also, deletes some files that were specific to the CMS Developer Portal, updates others so that we can use them for the Blue Button static site. Updated files include `ops/Jenkinsfile` to facilitate a simple deployment task on CBJ.

### CODE REVIEW

File: _config-imp.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-url: "https://imp.bluebutton.cms.gov"
-tealium_env: "dev"
```
Comment (samgensburg-gov) [inline _config-imp.yml]: Does this mean we're not using Tealium? I honestly don't know anything about it, but this has me wondering. If so, can we remove a couple of other remnants of this work?

File: ops/Jenkinsfile [hunk 2]
```diff
@@ -1,84 +1,97 @@
...
+def aws_creds = 'cbj-deploy'
+
 pipeline {
   agent any
+
   parameters {
-    string(name: 'GIT_TAG', defaultValue: '', description: '')
+    string(
+      name: 'GIT_VERSION',
+      defaultValue: '*/master',
```
Comment (samgensburg-gov) [inline ops/Jenkinsfile]: Do we want to consider using develop in lower environments? I'm not sure how to do this here, but my thinking was that develop would be used for the dev environment. This is also fine if you think this is more appropriate, I was just thinking we would eventually follow for the application repo

File: _config-imp.yml (hunk 1 above)
Comment (rnagle) [reply _config-imp.yml]: I don't know anything about it either. There are probably other left overs we can remove. I've just been scrubbing stuff that I think is unnecessary and is near whatever I'm working on at the moment.

File: ops/Jenkinsfile (hunk 2 above)
Comment (rnagle) [reply ops/Jenkinsfile]: I think this is a good idea. There's a little work that we'll need to do to clean up the develop branch before we can start using it. Should probably create a ticket to get this done at some point.

**Pull request merged**